### PR TITLE
Update utils.py

### PR DESCRIPTION
**Pull request by hemajv**
Setting the PROMETHEUS_PUSHGATEWAY_HOST AND PROMETHEUS_PUSHGATEWAY_PORT env vars (funct -run_analyzer) to expose the prometheus metrics when the analyzer job is executed

### CODE REVIEW

**Comment by fridex** [comment]
@hemajv LGTM, but please rebase on top of the current master

**Comment by durandom** [comment]
@goern @fridex can you give @hemajv some guidance on what fails here and what is missing? Those `linters` links dont work for me :( 
And what is `local/check — check status: failure`

File: thoth_user_api/utils.py [hunk 1]
```diff
@@ -154,6 +156,14 @@ def run_analyzer(image: str, analyzer: str, debug: bool = False,
             f"{registry_user}:{registry_password}"
         )
 
+
```
Comment (fridex) [inline thoth_user_api/utils.py]: Linter is complaining about the blank line you added - there are 2 blank lines not necessary, delete one.

Comment (durandom) [reply thoth_user_api/utils.py]: @goern @fridex is there a way to run that locally? 
I have a `pre-commit` [hook](https://github.com/durandom/dotfiles/tree/master/git_template/hooks) that runs rubocop local before committing. Maybe you want to add that to your git hook collection?

Comment (fridex) [reply thoth_user_api/utils.py]: https://github.com/thoth-station/core/blob/master/doc/development-good-practice.md#install-git-hooks-on-local

**Comment by fridex** [comment]
> Build failed.
>
> - [linters ](https://zuul.thoth-station.ninja/logs/27/127/06a947973d51b6a2e406f76c486b68db96b7e974/check/linters/5295c72/) : FAILURE in 36s


hm.. the error seems to be suspicious to me. @goern can we run coala with ``--no-color`` to suppress colored output? (just guessing what could go wrong)

**Comment by goern** [comment]
this seems to be the right thing: if _PROMETHEUS_PUSHGATEWAY_HOST and _PROMETHEUS_PUSHGATEWAY_PORT on the other PR you just check for host not for port

**Comment by hemajv** [comment]
@goern are you referring to [https://github.com/thoth-station/package-extract/pull/68] this PR?

**Comment by goern** [comment]
@hemajv  ja

**Pull request merged**